CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Changelog
 
+## 2022.7.100 (29 July 2022)
+
+### Enhancements
+
+1. Warn users when a Python file could override an existing Python package (there by interfering with the kernels. [More info](https://aka.ms/JupyterKernelStartFailureOverrideReservedName)).
+       This feature could be turned off via the setting `"jupyter.diagnostics.reservedPythonNames.enabled": false`.
+   ([#7538](https://github.com/Microsoft/vscode-jupyter/issues/7538))
+1. DataFrame viewer enabled on the web.
+   ([#9665](https://github.com/Microsoft/vscode-jupyter/issues/9665))
+1. The Variable Viewer now shows strings wrapped in single quotes.
+   ([#10225](https://github.com/Microsoft/vscode-jupyter/issues/10225))
+1. Rework kernel selection to be in either 'remote' mode or 'local' mode to avoid confusion about what kernels should be displayed.
+   ([#10435](https://github.com/Microsoft/vscode-jupyter/issues/10435))
+1. Added a more info button to the kernel depedency prompt to give more information about ipykernel.
+   ([#10658](https://github.com/Microsoft/vscode-jupyter/issues/10658))
+1. Use the Python Extension install python command versus just pointing at python.org.
+   ([#10696](https://github.com/Microsoft/vscode-jupyter/issues/10696))
+1. Only show the "kernel may need to be restarted" message when installing Python if there are active kernels.
+   ([#10697](https://github.com/Microsoft/vscode-jupyter/issues/10697))
+1. Added a setting 'jupyter.showOnlyOneTypeOfKernel' to allow trying out the new kernel picker UI.
+   ([#10782](https://github.com/Microsoft/vscode-jupyter/issues/10782))
+1. Don't show the Python getting started page when installing via the Jupyter kernel picker command.
+   ([#10793](https://github.com/Microsoft/vscode-jupyter/issues/10793))
+
+### Fixes
+
+1. Fixes problem where clipboard permissions are required in order to enter a Jupyter server URL.
```

Comment:
Turns out this fix isn't enabled unless the 'showOnlyOneTypeOfKernel' is selected. We might want to mention that in the changelog.